<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnModificarLote</class>
 <widget class="QWidget" name="vtnModificarLote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Lote</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Lote</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1" alignment="Qt::AlignLeft">
         <widget class="QDateEdit" name="dateFechVenc"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Número</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableWidget" name="tablaLote">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>160</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btnBuscarLote">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>* Fecha de Vencimiento</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineNumero">
          <property name="statusTip">
           <string>Ingrese número de lote (solo letras y números sin espacios)</string>
          </property>
          <property name="accessibleDescription">
           <string>lote</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>* Cantidad</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineCant">
          <property name="statusTip">
           <string>Ingrese cantidad del producto para dicho lote</string>
          </property>
          <property name="accessibleDescription">
           <string>cantidad</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Producto</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QPushButton" name="btnBuscarProd">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>* Código de Barra</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineCodBarra">
          <property name="statusTip">
           <string>Ingrese código de barra del producto (exactamente 9 números)</string>
          </property>
          <property name="accessibleDescription">
           <string>codigo</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableWidget" name="tablaProducto">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>160</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineNumero</tabstop>
  <tabstop>btnBuscarLote</tabstop>
  <tabstop>tablaLote</tabstop>
  <tabstop>dateFechVenc</tabstop>
  <tabstop>lineCant</tabstop>
  <tabstop>lineCodBarra</tabstop>
  <tabstop>btnBuscarProd</tabstop>
  <tabstop>tablaProducto</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
